src/Core/Domain/ServiceModel.ts

```diff
@@ -108,11 +108,12 @@ export interface InstanceSummary {
  */
 export interface ServiceModel extends ServiceIdentifier {
   environment: string;
-  description?: string;
+  description?: string | null;
   lifecycle: LifecycleModel;
   attributes: AttributeModel[];
   service_identity?: string;
-  service_identity_display_name?: string;
+  service_identity_display_name?: string | null;
```

Comment:
Can they be null? or just absent 🤔

Reply:
They can be null, that's what our mocks form local-setup have set up